### ISSUE TITLE
Not Support Android & IOS

### ISSUE THREAD

**Issue by hadow** [Eastrall/MonoGame-HardwareInstancing#1]
private void PlatformDrawInstancedPrimitives(PrimitiveType primitiveType, int baseVertex, int startIndex, int primitiveCount, int instanceCount)
        {
            throw new NotImplementedException("GraphicsDevice.DrawInstancedPrimitives is not yet implemented for OpenGL.");
        }

**Comment by Eastrall** [comment]
Yes, the `DrawInstancedPrimitives` method is only available for DirectX platform for now. 

I saw on MonoGame repository that this features is already done and should be shipped with version `3.7`.
